.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## PR Description
+
+Please include a summary of the change along with the relevant motivation and context.
+
+## Notion ticket
+
+Ticket link
+
+## Screenshots
+
+Please add screenshots for any new features or UI bug fixes for the following browsers -
+
+- Chrome
+  >
+- Firefox
+  >
+- Safari
+  >
+
+## Security
+
+- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.
```

Comment:
These are not strictly applicable to the SDK. I'd rather tweak the existing org template.

Reply:
lets set a meeting this week with  @MoumitaM and decide on what we want to place in PR template